/* Error Handling
--------------------
try-catch  (try catch block)


try{
    ...normal code
}catch(err){ //err is error object
    ...handling error
}

*/
